src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +61,58 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyIPATrust)
+def test_ipa_trusts__ipa_server_trusted_user_lookup(ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA server
+    :setup:
+        1. Restart SSSD and clear cache on IPA server
+    :steps:
+        1. Resolve trusted domain admin user
+    :expectedresults:
+        1. User is resolved
+    :customerscenario: True
+    """
+    ipa.sssd.clear(db=True, memcache=True, logs=True)
+    ipa.sssd.restart()
+
+    # Resolve user
+    username = trusted.admin_fqn()
+
+    id_user = ipa.tools.id(username)
+    assert id_user is not None
+    assert id_user.user.name == username
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__ipa_client_trusted_user_lookup(client: Client, ipa: IPA, trusted: IPA):
```

Comment:
Like all the other tests, we can assume it uses the client. 

test_ipa_trusts__lookup_a_trusted_user

Reply:
Changed to `test_ipa_trusts__lookup_trusted_user`